main.py

```diff
@@ -25,25 +31,62 @@ def value(self, value):
 
 
 class Name(Field):
-    pass
+    def __init__(self, n):
+        self.name = n
 
 
 class Phone(Field):
-    pass
-
+    def __init__(self, phone = None):
+        self.phone = phone
 
 class Birthday(Field):
-    pass
+     def __init__(self, birthday_y = None, birthday_m = None, birthday_d = None,):
```

Comment:
Lets change args for format("23.12.1996") input  and parse it using "try...except" for wrong input. As there should be additional checks then (like 65,23,1234)